database/user.go

```diff
@@ -2,33 +2,77 @@ package database
 
 import (
 	"context"
+	"strconv"
+	"sync"
+	"sync/atomic"
+	"time"
 
 	"github.com/NebulousLabs/skynet-accounts/build"
 
 	"gitlab.com/NebulousLabs/errors"
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
-// User status tiers.
 const (
+	// TierReserved reserved
 	TierReserved = iota
+	// TierFree free
 	TierFree
+	// TierPremium5 5
 	TierPremium5
+	// TierPremium20 20
 	TierPremium20
+	// TierPremium80 80
 	TierPremium80
+
+	// KiB kilobyte
+	KiB = 1024
+	// MiB megabyte
+	MiB = 1024 * KiB
+
+	// PriceBandwidthRegistryWrite the bandwidth cost of a single registry write
```

Comment:
Do you mean some exhaustive explanation on the entire pricing model? I agree that it would be beneficial. But maybe we can add it a little later, when we have it a little more stable. David introduced a new unit yesterday, so I'd say things are a little unstable, still.